src/window/building/common.c

```diff
@@ -70,9 +85,14 @@ void window_building_draw_employment(building_info_context *c, int y_offset)
     } else {
         text_id = 19; // too few people allocated
     }
-    if (!text_id && b->houses_covered < 40) {
+    if (!text_id && consider_house_covering && b->houses_covered < 40) {
         text_id = 20; // poor access to employees
     }
+    return text_id;
+}
+
+void draw_employment_details(building_info_context *c, int y_offset, int text_id, building *b)
```

Comment:
Make this function `static` and move it up.

Reply:
done